xml/operations-compute-using_flavor_metadata.xml

```diff
@@ -8,143 +8,29 @@
 <section xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" xml:id="topic_vhs_12v_vw">
  <title>Using Flavor Metadata to Specify CPU Model</title>
  <para>
-  In &kw-hos; Nova, the <literal>ComputeCapabilitiesFilter</literal>
+  In &kw-hos; &o_comp;, the <literal>ComputeCapabilitiesFilter</literal>
   scheduling filter matches the exact CPU model of the compute host and the
-  requested compute instance model. It doesn't entertain the libvirt's
-  capability of emulating the hosts CPU model to its guests if the guest's CPU
-  model is older or equal to that of the hosts.
+  requested compute instance model. It does not support the libvirt capability
+  of emulating the host CPU model to its guests if the guest's CPU model is
+  older or equal to that of the hosts.
  </para>
  <para>
-  To assist, we have introduced a new Nova scheduler filter which captures the
-  cpu_models as a subset of a particular CPU family. The filter determines if
-  the host CPU model is capable of emulating the guest CPU model by
-  maintaining the mapping of the vCPU models per vendor (currently Intel and
-  AMD) and compares it against the 'model' of the host.
+  To assist, there is a &o_comp; scheduler filter that captures the cpu_models
+  as a subset of a particular CPU family. The filter determines if the host CPU
+  model is capable of emulating the guest CPU model by maintaining the mapping
+  of the vCPU models per vendor (currently Intel and AMD) and compares it
+  against the 'model' of the host.
  </para>
  <para>
-  There is a current limitation which is when a particular cpu_model is
-  requested through <literal>hw:cpu_model</literal> via a compute flavor, the
-  cpu_mode will be set to "custom". This mode makes it so that a persistent
-  guest virtual machine will see the same hardware no matter what host
-  physical machine the guest virtual machine is booted on. This allows easier
-  live migration of virtual machines. Because of this limitation, only some of
-  the features of a CPU are exposed to the guest. Requesting a particular CPU
-  feature is not currently supported.
+  There is a limitation when a particular cpu_model is requested through
+  <literal>hw:cpu_model</literal> via a compute flavor, the cpu_mode will be
```

Comment:
Should the second occurrence at the end of the line really be "cpu_mode" or should it be "cpu_mode**l**" as well?

Reply:
"mode" is better given other content there. 
"...the cpu_mode will be
  set to <literal>custom</literal>. This *mode* makes it so that a persistent
  guest virtual machine..." The latter sentence doesn't make sense if "model" is used.

Reply:
Ah, you are right, I guess. https://wiki.openstack.org/wiki/LibvirtXMLCPUModel